src/MultiField/MultiFieldCellFields.jl

```diff
@@ -39,3 +39,167 @@ num_fields(a::MultiFieldCellField) = length(a.single_fields)
 Base.getindex(a::MultiFieldCellField,i::Integer) = a.single_fields[i]
 Base.iterate(a::MultiFieldCellField)  = iterate(a.single_fields)
 Base.iterate(a::MultiFieldCellField,state)  = iterate(a.single_fields,state)
+
```

Comment:
Why do you need to implement this here in `src/MultiField/MultiFieldCellFields.jl` ? I would say this should go in `src/CellData/CellFields.jl`. The implementation of the evaluation of a `CellField` in some `Points` is conceptually decoupled from FE spaces,multifield etc. It just needs the `CellField` interface and some of the upstream modules.